docker-for-azure/faqs.md

```diff
@@ -118,4 +118,8 @@ westeurope
 westindia
 westus
 westus2
+australiaeast
+australiasoutheast
+francecentral
+francesouth
```

Comment:
Could you keep the list sorted alphabetically?

Reply:
It's super complicated task, but I did it. :)

P.S. I don't have access to Azure in China or to US Gov/DoD, so I can't add all regions